src/Components/Web.JS/src/Platform/Circuits/DefaultReconnectDisplay.ts

```diff
@@ -52,7 +53,7 @@ export class DefaultReconnectDisplay implements ReconnectDisplay {
         // - true to mean success
         // - false to mean we reached the server, but it rejected the connection (e.g., unknown circuit ID)
         // - exception to mean we didn't reach the server (this can be sync or async)
-        const successful = await window['Blazor'].reconnect();
+        const successful = await (Blazor?.reconnect as any)();
```

Comment:
Sorry if I'm missing something obvious, but why is it `Blazor?.reconnect` here and not just `Blazor.reconnect`? AFAICT `Blazor` would always be populated here.

Reply:
Perhaps it's a typo. In `DefaultReconnectionHandler.ts`, there's no `?` on the equivalent.

Reply:
Also has it in `DefaultReconnectionHandler.ts`
```ts
this._reconnectCallback = reconnectCallback || (() => (Blazor.reconnect as any)());
```

Its because `Blazor` is created in `GlobalExports.ts` however `.reconnect` is assigned in `Boot.Server.ts` or `Boot.WebAssembly.ts` so it has to be nullable (due to late assignment); TypeScript then gives error "can't invoke method that may be undefined"; so need the `as any` cast to get around that compile error.